clients/tabby-agent/src/TabbyAgent.ts

```diff
@@ -1010,4 +1018,16 @@ export class TabbyAgent extends EventEmitter implements Agent {
       return null;
     }
   }
+
+  private clientInfoToString(session: Record<string, any> | undefined): string {
+    const nodeInfo = `Node.js/${process.version}`;
```

Comment:
We need to check `isBrowser` here. In browser environment, use `navigator?.userAgent` instead. Here is an [example](https://github.com/TabbyML/tabby/blob/main/clients/tabby-agent/src/AnonymousUsageLogger.ts#L16).